gocumber_test.go

```diff
@@ -201,3 +201,19 @@ func TestExec_NoMatch(t *testing.T) {
 
 	assert.False(t, steps.Exec("some unknown step"))
 }
+
+func TestDocstrings_OutlineVariables(t *testing.T) {
+	steps := make(Definitions)
+	tt := new(testing.T)
+
+	var called bool
+	steps.Given("something is:", func(_ []string, step StepNode) {
+		called = true
+		assert.Equal(t, "minimally functional\n\n", step.PyString().String())
```

Comment:
This has to do with how go-gherkin prints out the lines via the `String()` function: https://github.com/muhqu/go-gherkin/blob/e07b6f9917cc9d90dd0a5d5f9ab9a63c82efe045/nodes/nodes.go#L463-L469

In reality what is stored in with the single \n. It's only when we print it in the test that is shows up with two \n's.

I'm at a bit of a loss. It seems like when we execute the step we will have the correct single \n character. I could resolve this like so:

```go
pyString.AddLine(replace(strings.Trim(original.PyString().String(), "\n")))
```

But wouldn't I be removing the newline that will be present in the steps?